scripts/fiximports

```diff
@@ -1,17 +1,22 @@
 #!/usr/bin/env bash
 
-go_files() {
-    find . -type f -name \*.go -not -name \*_cbor_gen.go | grep -v './extern/filecoin-ffi' | grep -v './extern/test-vectors'
-}
-
-go_files | xargs -I '{}' sed -i.ifixbak -e '/import (/ {
+sed_replace='/import (/ {
   :1
   $!N
   s/\n\n/\'$'\n''/
   /)/!b1
-}' '{}'
+}'
+
+go_files() {
+    find . -type f -name \*.go -not -name \*_cbor_gen.go | grep -v './extern/filecoin-ffi' | grep -v './extern/test-vectors'
+}
 
-find . -type f -name \*.go.ifixbak | xargs rm
+# Because -i works differently on macOS, we need to use a different sed command
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    go_files | xargs -I '{}' sed -i '' -e "$sed_replace" '{}'
```

Comment:
Can we not just specify the suffix anyways?

Reply:
Oh, I see. Also, I absolutely hate macos.

Do we even need this? Can we just _not_ delete these lines?

Reply:
Ah... I see why it's there. It's there because we add a bunch of newlines and `goimports` won't _ungroup_.